pkg/install/squash.go

```diff
@@ -30,7 +26,7 @@ var (
 // ClusterRole - enabling pod creation
 // ClusterRoleBinding - bind ClusterRole to Squash's ServiceAccount
 // Deployment - Squash itself
-func InstallSquash(cs *kubernetes.Clientset, namespace, containerRepo, containerVersion string, preview bool) error {
+func InstallSquash(cs *kubernetes.Clientset, namespace, containerRepo, preview bool) error {
```

Comment:
containerRepo --> containerRepo string

Reply:
exactly, as @hgword said:
```suggestion
func InstallSquash(cs *kubernetes.Clientset, namespace, containerRepo string, preview bool) error {
```